google/cloud/spanner_v1/_pandas_helpers.py

```diff
@@ -0,0 +1,76 @@
+import logging
+import six
+
+class pandas_df():
+    
+    def pd_dataframe(self):
+        """Returns the response in a pandas dataframe of the StreamedResultSet object"""
+        try :
+            from pandas import DataFrame
+        except ImportError:
+            logging.error("Pandas module not found. It is needed for converting query result to Dataframe.\n Try running 'pip3 install pandas'")
+
+        code_to_spanner_dtype_dict = {
+                                        1 : 'BOOL',
```

Comment:
Don't use the raw integers. If you "view source" for TypeCode https://googleapis.dev/python/spanner/latest/spanner_v1/types.html#google.cloud.spanner_v1.types.TypeCode you'll notice that there are several defined constants which can be used instead.

Reply:
Done